ui/core/src/model/units/bytes.ts

```diff
@@ -32,17 +40,14 @@ export const BYTES_GROUP_CONFIG: UnitGroupConfig = {
   abbreviate: true,
 };
 export const BYTES_UNIT_CONFIG: Readonly<Record<BytesUnitKind, UnitConfig>> = {
-  // These units are powers of 1000.
-  // In other words, 1 KB = 1000 bytes.
   Bytes: {
     group: 'Bytes',
     label: 'Bytes',
   },
 };
 
-export function formatBytes(bytes: number, options: BytesUnitOptions) {
-  const { abbreviate, decimal_places } = options;
-
+export function formatBytes(bytes: number, { abbreviate, decimal_places }: BytesUnitOptions) {
```

Comment:
Destructuring parameters inside the arguments themselves -- for consistency with other files